models/issue.go

```diff
@@ -1543,7 +1543,7 @@ func CountIssues(opts *IssuesOptions) (int64, error) {
 		return 0, fmt.Errorf("Find: %v", err)
 	}
 	if len(countsSlice) < 1 {
```

Comment:
Alternatively, we could check for `len(countsSlice) == 0`.
Which do you prefer?

Reply:
I think it does not matter - but I'll change it as you are right - this is strange